README.md

```diff
@@ -8,6 +8,15 @@ To use the ArDrive CLI, install it with your favorite package manager, and run "
 
 ## Setup
 
+### NVM
+
+This repo uses NVM and an `.nvmrc` file to lock the Node version to the current version used by `ardrive-core-js`. Follow these steps to get NVM up and running on your system:
+
+1. Install NVM using [these installation instructions][nvm-install].
+2. Navigate to this project's root directory (`ardrive-cli`)
```

Comment:
Specifying that `ardrive-cli` is the root folder name might be presumptuous; users can choose any root folder name for a git repo checkout.